Sources/PackageLoading/ModuleMapGeneration.swift

```diff
@@ -24,6 +24,24 @@ extension CModule {
 
 extension ClangModule {
     
+    /// A link-declaration specifies a library or framework
+    /// against which a program should be linked.
+    /// More info: http://clang.llvm.org/docs/Modules.html#link-declaration
+    public enum ModuleMapLinkDecl {
```

Comment:
This should be a "style", framework or library, since it impacts both the kind of module map and the link declaration.